apps/src/templates/SignInOrAgeDialog.js

```diff
@@ -20,6 +20,11 @@ const styles = {
     fontSize: 16,
     fontFamily: "'Gotham 5r', sans-serif",
   },
+  dancePartyHeading: {
+    fontSize: 32,
+    fontWeight:'bold',
+    fontFamily: "'Gotham 5r', sans-serif",
```

Comment:
You can use `fontFamily: "'Gotham 7r', sans-serif"` instead of the additional style of `fontWeight:'bold'`, like here: https://github.com/code-dot-org/code-dot-org/blob/47379314467b1bad950908600ef9473b893dd8e7/apps/src/templates/studioHomepages/ResourceCard.jsx#L24